adapter.go

```diff
@@ -79,15 +82,15 @@ func NewAdapter(url string, timeout ...interface{}) (persist.BatchAdapter, error
 	if connString.Database != "" {
 		databaseName = connString.Database
 	} else {
-		databaseName = "casbin_rule"
+		databaseName = defaultDatabaseName
 	}
 
-	return NewAdapterWithClientOption(clientOption, databaseName, timeout...)
+	return NewAdapterWithClientOption(clientOption, databaseName, defaultCollectionName, timeout...)
 }
 
 // NewAdapterWithClientOption is an alternative constructor for Adapter
 // that does the same as NewAdapter, but uses mongo.ClientOption instead of a Mongo URL
-func NewAdapterWithClientOption(clientOption *options.ClientOptions, databaseName string, timeout ...interface{}) (persist.BatchAdapter, error) {
+func NewAdapterWithClientOption(clientOption *options.ClientOptions, databaseName string, collectionName string, timeout ...interface{}) (persist.BatchAdapter, error) {
```

Comment:
Don't make breaking change. You can add new constructor for your new parameter.

Reply:
fixed.

the change of the default database name to `casbin` not `casbin_rule` might create an issue as well?
(although the db name must be "casbin" not "casbin_rule" as mentioned in the comment above )

Reply:
@0xb4lamx I think the DB name change is also a breaking change. So we have to make a major version release.

Reply:
Can you also add the 2 new constructors to our README?

Reply:
done. ready to merge.